pkgs/top-level/perl-packages.nix

```diff
@@ -12122,6 +12122,25 @@ let self = _self // overrides; _self = with self; {
     };
   };
 
+  PerconaToolkit = buildPerlPackage rec {
+    name = "Percona-Toolkit-3.0.11";
+    src = fetchFromGitHub {
+      owner = "percona";
+      repo = "percona-toolkit";
+      rev = "${version}";
```

Comment:
Maybe I am missing it, but where do you declare variable `version`. I only see `name = "Percona-Toolkit-3.0.11";`.

Reply:
Fixed.